mlflow/store/sqlalchemy_store.py

```diff
@@ -19,9 +19,35 @@
 
 
 class SqlAlchemyStore(AbstractStore):
+    """
+    SQLAlchemy compliant backend store for tracking meta data for MLflow entities. Currently
+    supported database types are ``mysql``, ``mssql``, ``sqlite``, and ``postgresql``. This store
+    interacts with SQL store using SQLAlchemy abstractions defined for MLflow entities.
+    :py:class:`mlflow.store.dbmodels.models.SqlExperiment`,
+    :py:class:`mlflow.store.dbmodels.models.SqlRun`,
+    :py:class:`mlflow.store.dbmodels.models.SqlTag`,
+    :py:class:`mlflow.store.dbmodels.models.SqlMetric`, and
+    :py:class:`mlflow.store.dbmodels.models.SqlParam`.
+
+    Run artifacts are stored in a separate location using artifact stores conforming to
+    :py:class:`mlflow.store.artifact_repo.ArtifactRepository`. Default artifact locations for
+    user experiments are stored in the database along with metadata/ Each run artifact location
+    is recorded in :py:class:`mlflow.store.dbmodels.models.SqlRun` and stored in the backend DB.
+    """
     ARTIFACTS_FOLDER_NAME = "artifacts"
 
     def __init__(self, db_uri, default_artifact_root):
+        """
+        Create a database backed store.
+
+        :param db_uri: SQL connection string used by SQLAlchemy Engine to connected to database.
+                       Argument is expected to in this format:
```

Comment:
Wording suggestion: `"Argument must follow the format database_type://<user_name>:<password>@<host>:<port>/<database_name>"`

Alternatively, add a "be" here (like "Argument is expected to be in the format:")